### PR TITLE
More updates to /poll

### PR DESCRIPTION
**Pull request by bumbadadabum**

### CODE REVIEW

**Comment by bumbadadabum** [comment]
woah some weird shit happened with the commit.

File: chat-plugins/poll.js [hunk 1]
```diff
@@ -152,12 +151,21 @@ exports.commands = {
...
 		'': function (target, room, user) {
 			if (!room.poll) return this.errorReply("There is no poll running in this room.");
 			if (!this.canBroadcast()) return;
+			room.update();
 
 			room.poll.display(user, this.broadcasting);
 		}
-	}
+	},
+	pollhelp: ["Poll plugin for Pokémon Showdown:",
```
Comment (Slayer95) [inline chat-plugins/poll.js]: No need to talk about plugins to users

File: chat-plugins/poll.js [hunk 2]
```diff
@@ -152,12 +151,21 @@ exports.commands = {
...
+				"This plugin allows rooms to run their own polls. These polls are limited to one poll at a time per room, and display to the room when they're created, ended, or when someone joins the room.",
+				"The poll is displayed in a textbox, and is updated in real time. It's also possible to vote by clicking the buttons on the poll itself.",
+				"This plugin contains the following commands:",
+				"/poll new [name] | [options] - Starts a new poll. The amount of options can be between 1 and 8. Requires: % @ # & ~",
```
Comment (Slayer95) [inline chat-plugins/poll.js]: The lowest limit is kind of obvious. Just say ``Limited to up to 8 options``.

File: chat-plugins/poll.js [hunk 3]
```diff
@@ -152,12 +151,21 @@ exports.commands = {
...
+				"/poll vote [number] - Votes for option [number] in the poll.",
```
Comment (Slayer95) [inline chat-plugins/poll.js]: `"in the poll"` is probably unneeded.

File: chat-plugins/poll.js (hunk 2 above)
Comment (panpawn) [inline chat-plugins/poll.js]: Just tested it, it actually doesn't even let you create a poll with 1 option anyways, so yea I like Slayer's suggestion.

File: chat-plugins/poll.js [hunk 4]
```diff
@@ -152,12 +151,21 @@ exports.commands = {
...
+			room.update();
 
 			room.poll.display(user, this.broadcasting);
 		}
-	}
+	},
+	pollhelp: ["Poll plugin for Pokémon Showdown:",
+				"This plugin allows rooms to run their own polls. These polls are limited to one poll at a time per room, and display to the room when they're created, ended, or when someone joins the room.",
+				"The poll is displayed in a textbox, and is updated in real time. It's also possible to vote by clicking the buttons on the poll itself.",
+				"This plugin contains the following commands:",
```
Comment (Slayer95) [inline chat-plugins/poll.js]: Too verbose. Also "plugin".

File: chat-plugins/poll.js [hunk 5]
```diff
@@ -152,12 +151,21 @@ exports.commands = {
...
 			if (!room.poll) return this.errorReply("There is no poll running in this room.");
 			if (!this.canBroadcast()) return;
+			room.update();
 
 			room.poll.display(user, this.broadcasting);
 		}
-	}
+	},
+	pollhelp: ["Poll plugin for Pokémon Showdown:",
+				"This plugin allows rooms to run their own polls. These polls are limited to one poll at a time per room, and display to the room when they're created, ended, or when someone joins the room.",
```
Comment (Slayer95) [inline chat-plugins/poll.js]: `and display to the room when they're created, ended, or when someone joins the room` <- I don't think we need to state this

File: chat-plugins/poll.js [hunk 6]
```diff
@@ -152,12 +151,21 @@ exports.commands = {
...
+				"The poll is displayed in a textbox, and is updated in real time. It's also possible to vote by clicking the buttons on the poll itself.",
```
Comment (Slayer95) [inline chat-plugins/poll.js]: Most of this is superfluous.

Just ``The poll status is displayed to the users and updated in real time``

File: chat-plugins/poll.js (hunk 2 above)
Comment (Zarel) [inline chat-plugins/poll.js]: Documentation should be as terse as possible. "Starts a new poll, 2–8 options"

**Pull request merged**